apis/swagger.yml

```diff
@@ -517,22 +494,108 @@ definitions:
       ID:
         type: "string"
         description: "ID of the created task."
-  
+      FileLength:
+        type: "integer"
+        description: |
+          The length of the file dfget requests to download in bytes.
+        format: int64
+      PieceSize:
```

Comment:
The `format` of `TaskCreateResponse.PieceSize` is `int64`. It's not consistent with `TaskInfo.PieceSize`'s format: `int32`.